packages/react/src/drafts/TabPanels/TabPanels.features.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import type {Meta} from '@storybook/react'
+import TabPanels from './TabPanels'
+import type {ComponentProps} from '../../utils/types'
+import {Button} from '../../Button'
+
+export default {
+  title: 'Drafts/Components/TabPanels/Features',
+  component: TabPanels,
+} as Meta<ComponentProps<typeof TabPanels>>
+
+export const Selected = () => (
+  <TabPanels aria-label="TabPanels example" id="tab-panels" selectedTabIndex={1}>
```

Comment:
For the controlled case, what's the intended way to synchronize the current selected tab with the index being provided? It seems like `onTabContainerChange` provides the tab but not the index so would that mean we would need to look it up in some way?

Reply:
@keithamus how would you feel about adding the tabIndex to the event object that gets raised?

Reply:
The "look it up in some way" is `event.target.selectedTabIndex`. But I'm fine to expose it as `event.selectedTabIndex`.